includes/discovery/sensors/states/trapeze.inc.php

```diff
@@ -0,0 +1,57 @@
+<?php
+//
+// Discovery of Power supplies on Juniper Wireless (Trapeze) devices.
+//
+if ($device['os'] == 'trapeze') {
+    $temp = snmpwalk_cache_multi_oid($device, 'trpzSysPowerSupplyEntry', array(), 'TRAPEZE-NETWORKS-SYSTEM-MIB', 'trapeze');
+    $cur_oid = '.1.3.6.1.4.1.14525.4.8.1.1.13.1.2.1.2.'; //trpzSysPowerSupplyStatus
+    if (!empty($temp)) {
+        //Create State Index
+        $state_name = 'trpzSysPowerSupplyStatus';
+        $state_index_id = create_state_index($state_name);
+        /*
+        Create State Translation
+
+        TrpzSysPowerSupplyStatus
+            INTEGER {
+                    other        (1),
+                    unknown      (2),
+                    ac-failed    (3),
+                    dc-failed    (4),
+                    ac-ok-dc-ok  (5)
+                }
+
+        The LibreNMS generic states is derived from Nagios:
```

Comment:
Can get rid of the comments about nagios derived status, we have that in the docs.